tests/unit_tests/constraint/test_fcialg.py

```diff
@@ -618,7 +640,9 @@ def test_fci_spirtes_example(self):
         assert nx.is_isomorphic(skel_graph.to_undirected(), expected_pag.to_undirected())
         assert set(expected_pag.edges()) == set(pag.edges())
 
-    def test_fci_complex(self):
+    @pytest.mark.parametrize("skeleton_method", [SkeletonMethods.NBRS, SkeletonMethods.COMPLETE])
+    @pytest.mark.parametrize("pds_skeleton_method", [SkeletonMethods.PDS])
+    def test_fci_complex(self, skeleton_method, pds_skeleton_method):
```

Comment:
I.e. here, we can add an argument for `selection_bias` and add a pytest parametrization

`@pytest.mark.parametrize('selection_bias', [True, False])`